xinference/model/llm/pytorch/core.py

```diff
@@ -33,20 +35,6 @@
 logger = logging.getLogger(__name__)
 
 
-class PytorchGenerateConfig(TypedDict, total=False):
```

Comment:
Let the server decide what the generate config should be. It is OK to have a duplicated definition on the client side.